.gitlab/helper_functions.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+CYAN="\e[0;36m"
+CLEAR="\e[0m"
```

Comment:
I would put `CLEAR` at the start or the end of this list.

Reply:
Couldn't be in the end since we use it in line 5